ethereum_history_api/circuits/lib/src/rlp/decode_test.nr

```diff
@@ -243,3 +243,65 @@ mod test_get_small_string_offset_and_length {
         let _ = get_small_string_offset_and_length(header_of_list_type);
     }
 }
+
+mod test_decode_to_rlp_header {
+    use crate::rlp::decode::{decode_to_rlp_header, RlpHeader, STRING, LIST};
+
+    #[test]
+    fn success_single_byte() {
+        let rlp = [0x00];
+        let expected = RlpHeader { offset: 0, length: 1, data_type: STRING };
+        assert(decode_to_rlp_header(rlp) == expected);
+    }
+
+    #[test]
+    fn success_short_string() {
+        let rlp = [0x83, 0x63, 0x61, 0x74];
```

Comment:
Could we add comments - what this data actually is like we have in tx and receipt rlp tests?
I know test name already describes it, but there is a difference between "short string" and "0x636174"

Reply:
Like this:
```
let list_with_single_string = [0xc1, 0x80]; // [""]
```